client/js/App.js

```diff
@@ -1,15 +1,31 @@
 import React, {Component} from 'react';
-import {View, Text, SafeAreaView} from 'react-native';
 import Login from './screens/Login';
 import RootStackNavigator from './navigation/RootStackNavigator';
-import {ApolloProvider} from '@apollo/react-common';
-import client from './config/api';
+import Event from './screens/Event';
+import UserProvider from './context/UserProvider';
+import client from './apollo';
+import {ApolloProvider} from 'react-apollo';
 
+const comments = [
+  {
+    user: 'Britney Spears',
+    comment:
+      'Glad to join the campaign. Love it so much. It inspired me to see the world clearer.',
+    date: 'October 25, 2019',
+  },
+  {
+    user: 'Kelly Clarkson',
+    comment: 'Really like trees',
+    date: 'October 26, 2019',
+  },
+];
```

Comment:
remove this mock data if not being used.

Reply:
Its still being used